packages/typespec-ts/test/azureIntegration/encodeDatetime.spec.ts

```diff
@@ -210,7 +210,7 @@ describe("EncodeDatetimeClient Rest Client", () => {
         .path(`/encode/datetime/responseheader/unix-timestamp`)
         .get();
       assert.strictEqual(result.status, "204");
-      assert.strictEqual(result.headers.value, "1686566864");
+      assert.equal(result.headers.value, 1686566864);
```

Comment:
why equal not strictEqual?

Reply:
This is a test case issue in server side https://github.com/microsoft/typespec/blob/main/packages/http-specs/specs/encode/datetime/mockapi.ts#L271, could you have a pr to update it as number?

Reply:
https://github.com/microsoft/typespec/pull/6423